src/ngraph/pass/nop_elimination.cpp

```diff
@@ -134,62 +134,230 @@ static bool eliminate_reshape_v1(const std::shared_ptr<Node>& node)
     return false;
 }
 
+static bool replace_squeeze_unsqueeze(const std::shared_ptr<Node>& node)
+{
+    auto shape_ps = node->get_output_partial_shape(0);
+    if (!shape_ps.is_dynamic() && shape_size(shape_ps.get_shape()) != 0)
+    {
+        shared_ptr<Node> reshape;
+        auto input = node->input_value(0).get_node_shared_ptr();
+        auto shape = shape_ps.get_shape();
+        std::vector<int64_t> vi;
```

Comment:
You can merge lines 145 and 146 into one:
`std::vector<int64_t> vi(shape.begin(), shape.end());`